patroni/api.py

```diff
@@ -581,8 +960,29 @@ def is_failover_possible(self, cluster: Cluster, leader: str, candidate: str, ac
         return action + ' is not possible: no good candidates have been found'
 
     @check_access
-    def do_POST_failover(self, action: Optional[str] = 'failover') -> None:
-        """Handles manual failovers/switchovers, mainly from "patronictl"."""
+    def do_POST_failover(self, action: str = 'failover') -> None:
+        """Handle a ``POST`` request to ``/failover`` path.
+
+        Handles manual failovers/switchovers, mainly from ``patronictl``.
+
+        The request body should be a JSON dictionary, and it can contain the following keys:
+            * ``leader``: name of the current leader in the cluster;
+            * ``candidate`` or ``member``: name of the Patroni node to be promoted;
```

Comment:
The `member` is kept only for backward compatibility.
```suggestion
            * ``candidate``: name of the Patroni node to be promoted;
```